### PR TITLE
Fix a few conversion factors - see #303

### PR DESCRIPTION
**Pull request by dr-shorthair**
Can't find reliable value for Tons of Refrigeration, or the NAT-based units

### CODE REVIEW

**Comment by steveraysteveray** [comment]
Our current entry for unit:TON_FG has two references. Not good enough?

**Comment by dr-shorthair** [comment]
The Wikipedia entry does not give a good conversion factor. 
OM is a bit better. I'll use that one.

**Comment by dr-shorthair** [comment]
Fixes #303

**Review by steveraysteveray — CHANGES_REQUESTED** [review]
Simon,
Thanks so much for getting into this and fixing various conversion factors. In reviewing your work, some pre-existing errors have come to light. Specifically, QUDT has interpreted the combination of the Kilo, Mega, Peta, etc. prefixes with binary units (Bit, Byte) to mean powers of 2, such as Kilo meaning 1024, etc. Assuming we continue with this approach, there seem to be some inconsistencies, such as the conversionMultiplier for PetaByte being exactly 1000 times larger than the one for TeraBYTE.
Would you mind reviewing and fixing the various BIT and BYTE units to ensure they are all consistently powers of 2 rather than powers of 10?

Meanwhile, we can consider whether we want to augment the vocabulary with something like KiloBYTE_decimal for the 1000x version that many people also use.

**Comment by dr-shorthair** [comment]
Yes - I struggled a bit with these. 
First: do we want to use NAT as the reference unit for information?

**Comment by steveraysteveray** [comment]
Yes, the documentation indicates that the SI unit is indeed the NAT, so that looks good.

**Review by jhodgesatmb — APPROVED** [review]

**Comment by dr-shorthair** [comment]
Now I've added the 'binary' equivalents all the way up to 'exbi'. 
However, I've followed https://en.wikipedia.org/wiki/Byte#Multiple-byte_units and used the binary multiplier 1024, 1024^3, 1024^6 for kilo, mega and giga. 

Also see https://github.com/qudt/qudt-public-repo/issues/303#issuecomment-749894875

Should be good to go now.

**Comment by dr-shorthair** [comment]
unit:ExaBYTE 5545177444479562475.3378569716654
unit:ExbiBYTE 6393154322601327829.8943153498712
unit:GibiBYTE 5954088943.6391441429912255610071
unit:GigaBYTE 5954088943.6391441429912255610071
unit:KibiBYTE 5678.2617031470719747459655389854
unit:KiloBYTE 5678.2617031470719747459655389854
unit:MebiBYTE 5814539.984022601702139868711921
unit:MegaBYTE 5814539.984022601702139868711921
unit:PebiBYTE 6243314768165359.2088811673338586
unit:PetaBYTE 5545177444479562.4753378569716654
unit:TebiBYTE 6096987078286.4836024230149744713
unit:TeraBYTE 5545177444479.5624753378569716654

File: vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl [hunk 1]
```diff
@@ -5544,6 +5544,21 @@ unit:EuropeanUnitOfAccount9
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "European Unit of Account 9 (E.U.A.-9) (Bonds market unit)"@en ;
 .
+unit:ExaBYTE
+  a qudt:Unit ;
+  dcterms:description "The exabyte is a multiple of the unit byte for digital information. The prefix exa means 10^18 in the International System of Units (SI), however, in information quantities kilo=1024 so ExaByte is 1024^6 Bytes."^^rdf:HTML ;
```
Comment (steveraysteveray) [inline vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:5549]: This description needs to be adjusted to state that ExaBYTE is 10^18 Bytes and not 1024^6 Bytes.

File: vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl [hunk 2]
```diff
@@ -9395,7 +9450,8 @@ unit:KiloBYTE-PER-SEC
   a qudt:DerivedUnit ;
   a qudt:Unit ;
   dcterms:description "A kilobyte per second (kByte/s) is a unit of data transfer rate equal to 8,000 bits per second, or 1,000 bytes per second, or 8 kilobits per second."^^rdf:HTML ;
```
Comment (steveraysteveray) [inline vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:9452]: Need to change to say 1024 bytes per second or 8096 bits per second.

**Review by steveraysteveray — CHANGES_REQUESTED** [review]
See just the few in-line comments that need fixing, related to the dcterms:description fields.
Almost there!

File: vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl (hunk 2 above)
Comment (dr-shorthair) [reply vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:9452]: Changed per this review

**Review by steveraysteveray — APPROVED** [review]

**Comment by steveraysteveray** [comment]
Whoops. More pesky merge conflicts!

**Comment by dr-shorthair** [comment]
Conflicts resolved

**Pull request merged**